app/models/data_resources/news_item.rb

```diff
@@ -135,4 +135,5 @@ def remove_emojis
 #  title                      :string(255)
 #  visible                    :boolean          default(TRUE)
 #  push_notifications_sent_at :datetime
+#  payload                    :text(65535)
```

Comment:
fehlt hier nicht noch ein `serialize: :json`?

Reply:
i thought the same, but the others also don't have that

Reply:
is there a JSON.parse somewhere else ? If not, i would store it as JSON serialized

Reply:
no, currently the main server is not using the data.
it looks well in the database and when requested per GraphQL:

|database|GraphQL|
|---|---|
|<img width="128" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-mainserver/assets/1942953/35f3caec-9d08-4c06-acba-0b0eab09c9d4">|<img width="160" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-mainserver/assets/1942953/79d18309-b2d1-433d-b47a-fe457f290d32">|

i would not add something in this scope now, because it would affect all the payloads in all models and we would need further testings.

Reply:
maybe `GraphQL::Types::JSON` ist Parsing it on the fly